app/js/Routes.jsx

```diff
@@ -8,18 +8,16 @@
  */
 import React, { Component } from 'react';
 import { Router, Route, Link, IndexRoute, hashHistory, browserHistory } from 'react-router';
-import App from './components/App';
-import HomePage from './components/homePage/HomePage';
-import ManageApps from './components/homePage/manageApps/ManageApps';
-import ManageSettings from './components/homePage/ManageSettings';
+import App from './components/App.jsx';
```

Comment:
Currently across the codebase, we have a trend of importing modules without the extension, I would recommend we use a `consistent import style`. If you are going to import modules with their extensions I think you should ensure all other import statements in the codebase have extensions and vice versa.

Reply:
There's no such trend @andela-jomadoye
If you take a closer look, you'll see that I made it consistent across the whole codebase.
Thank you

Reply:
Noted @kingisaac95.